docs/how_tos/modifying_base_container.rst

```diff
@@ -0,0 +1,40 @@
+========================================
+Modifying the Base Container in Authn
+========================================
+
+The base container in Authn serves as the fundamental layout structure for rendering different components based on configurations. This document outlines the process for modifying the base container to accommodate changes or customize layouts as needed.
+
+Understanding Base Container Versions
+--------------------------------------
+
+The base container supports two main versions:
+
+- **Default Layout:** The default layout is the standard layout used when specific configurations do not dictate otherwise.
+.. image:: ./docs/images/default_layout.png
+- **Image Layout:** The image layout is an alternative layout option that can be enabled based on configurations.
+.. image:: ./docs/images/image_layout.png
```

Comment:
Images are not visible. looking like
<img width="1040" alt="Screenshot 2024-02-09 at 12 19 08 PM" src="https://github.com/openedx/frontend-app-authn/assets/52817156/1495b521-eb37-43ca-86c7-d3881fae53ea">